### PR TITLE
Fix alphaPicker padding 

### PR DESCRIPTION
**Pull request by grafixeyehero**
**Issues**
alphaPicker: page padding on the wrong side in TV mode #553

Tv mode
TV-mode

Desktop mode
Desktop-mode

Mobile mode
Mobile-mode

### CODE REVIEW

**Comment by dkanada** [comment]
Is there any particular design standard that requires the picker to be on the left for TV layouts? Maybe we can just remove this logic entirely.

**Comment by dkanada** [comment]
Somewhat related but can you figure out how to center this vertically next to the list?

a

**Comment by JustAMan** [comment]
> Maybe we can just remove this logic entirely.

Yeah, maybe we should just show picker on the right (or on the left)?
It being on the left might be because it was intended to be selectable by D-pad remote?..

**Comment by JustAMan** [comment]
cc @dmitrylyzo @EraYaN for TV insights

**Comment by dmitrylyzo** [comment]
Personaly, I'm used to having this on the right, because I look on page from left to right, and alphaPicker is not "distracting".

Also, if the left menu will be (_someday_) navigable with keyboard and should be on library page, then right alphaPicker is preferable.

**Comment by JustAMan** [comment]
In this case let's move it to the right on all layouts I think.

**Comment by grafixeyehero** [comment]
@dkanada the alphapicker it's already centered or did i miss some think

Annotation 2020-01-22 185027

**Comment by EraYaN** [comment]
I think they mean the filter and sort buttons.

Also yes it is selectable with the D-Pad, although the position of it is very stupid. But the right seems like the popular choice.

**Comment by dkanada** [comment]
I meant that on large portrait screens, the alpha picker isn't centered vertically next to the item container as seen above.

**Comment by grafixeyehero** [comment]
it happen when you take the screen shot the whole page

File: src/controllers/movies/movies.js [hunk 1]
```diff
@@ -169,11 +169,8 @@ define(["loading", "layoutManager", "userSettings", "events", "libraryBrowser",
                     valueChangeEvent: "click"
                 });
 
-                if (layoutManager.desktop || layoutManager.mobile) {
-                    alphaPickerElement.classList.add("alphabetPicker-right");
-                    itemsContainer.classList.remove("padded-left-withalphapicker");
-                    itemsContainer.classList.add("padded-right-withalphapicker");
-                }
+                alphaPickerElement.classList.add("alphaPicker-fixed-right");
+                itemsContainer.classList.add("padded-right-withalphapicker");
```
Comment (dkanada) [inline src/controllers/movies/movies.js]: Does this one not need the `tabContent` object used in the other files?

**Pull request merged**